server/src/test/java/org/opensearch/transport/InboundDecoderTests.java

```diff
@@ -65,36 +75,15 @@ public void testDecode() throws IOException {
         long requestId = randomNonNegativeLong();
         final String headerKey = randomAlphaOfLength(10);
         final String headerValue = randomAlphaOfLength(20);
+        final BytesReference totalBytes;
         if (isRequest) {
             threadContext.putHeader(headerKey, headerValue);
+            totalBytes = serialize(isRequest, Version.CURRENT, false, false, action, requestId, new TestRequest(randomAlphaOfLength(100)));
```

Comment:
You don't need this conditional logic for `serialize` - these calls are identical,  just use it the same way after `if` block (https://github.com/opensearch-project/OpenSearch/pull/13293/files#r1593249748)